partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs

```diff
@@ -126,6 +129,31 @@ fn run_pc_contracts_cli_addresses<C: IOContext>(
 			.ok_or(anyhow!("Permissioned candidates policy id from pc-contracts-cli addresses command output cannot be converted to string"))?
 			.to_string(),
 		context);
+	ILLIQUID_SUPPLY_ADDRESS.save_to_file(
+		&addresses_json.pointer("/addresses/IlliquidCirculationSupplyValidator")
+			.ok_or(anyhow!("Illiquid circulation supply validator address is missing from pc-contracts-cli addresses command output"))?
+			.as_str()
+			.ok_or(anyhow!("Illiquid circulation supply validator address from pc-contracts-cli addresses command output cannot be converted to string"))?
+			.to_string(),
+		context,
+	);
+	Ok(())
+}
+
+fn prepare_native_token<C: IOContext>(context: &C) -> anyhow::Result<()> {
+	context.print(
+		"Partner Chains can store their initial token supply on Cardano as Cardano native tokens.",
+	);
+	context.print("Creation of the native token is not supported by this wizard and must be performed manually before this step.");
```

Comment:
Is there a plan, like in JIRA, to have this supported?
What should be the flow? At what point is policy_id/asset_name chosen? Does it involve using smart-contracts CLI?

Reply:
I talked with Piotr about this, and he told me we don't want to support this at all, at least for the time being. The issue I see with doing that, is that the token policy can have custom logic in it, so whatever we provide would be relevant only to those PC devs that want a token with exactly the logic that we provide (presumably none).

Reply:
> At what point is policy_id/asset_name chosen? Does it involve using smart-contracts CLI?

No, it can be any native token on Cardano. One can even be created using an existing creator, but someone can roll their own token code with custom logic and deploy it themselves